geomesa-spark/geomesa-spark-jts/src/main/scala/org/locationtech/geomesa/spark/jts/util/GeoHashUtils.scala

```diff
@@ -0,0 +1,165 @@
+/***********************************************************************
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.spark.jts.util
+
+import com.vividsolutions.jts.geom.{Envelope, Geometry, GeometryFactory, PrecisionModel}
+import scala.collection.BitSet
+import scala.collection.immutable.{BitSet => IBitSet}
+
+object GeoHashUtils {
```

Comment:
should add some comments differentiating this from the other GeoHashUtils in geomesa-util, e.g. why this one, differences between the two, etc